common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -0,0 +1,752 @@
+"""
+Bok choy acceptance and a11y tests for problem types in the LMS
+
+See also lettuce tests in lms/djangoapps/courseware/features/problems.feature
+"""
+import random
+import textwrap
+
+from abc import ABCMeta, abstractmethod
+from nose.plugins.attrib import attr
+from selenium.webdriver import ActionChains
+
+from capa.tests.response_xml_factory import (
```

Comment:
Interesting. Do we use this approach in other tests? I don't have a firm position on it, but am curious since we're crossing over into unit test fixtures.